[12.0] Feature - Contábil e Financeiro

mileo opened the pull request:
- [x] Modelo de Plano de contas para empresas do regime normal;
- [x] Modelos de lançamentos contábeis;
- [x] Histórico padrão de lançamento contábil;
- [x] Mixin de pagamentos / recebimentos;
- [x] Módulo financeiro integrado com a contabilidade;
- [x] Definição de uma forma de não duplicar dados presentes em l10n_br_simple e l10n_br_cfc


#69

mileo committed:
[NEW] Modelos de partidas dobradas genérico

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

gabrielcardoso21 committed:
[ADD] Domain of field_id

gabrielcardoso21 committed:
[FIX] View error

mileo committed:
[NEW] Módulo de relatórios contabeis

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[NEW] Some fixes and report styles

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[NEW] Move history

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[FIX] Move template access

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[IMP] Move template and template lines

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[NEW] Template at fiscal_opereration_line

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[NEW] Template at document and document line

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[NEW] Contabilização de Operações de Vendas

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[NEW] Generate account moves from l10n_br_fiscal

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[NEW] Operation data

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo commented:
![image](https://user-images.githubusercontent.com/1975978/78651139-4e29a100-7896-11ea-98fb-28e57dfe7e9c.png)

mileo committed:
[REF] Comment engine

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[REF] move history comment engine

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[IMP] Move template date

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[REM] Legacy unicode code from v10

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[NEW] History on account move template

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[FIX] Already computed fields

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[NEW] Avoid invoice action_move_create

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[NEW] Payment and Payment Lines

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[NEW] Generate financial move lines

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

renatonlima committed:
rename file

mileo committed:
[COPY] account.payment.term

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[REF] Split files

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo commented:
Depende do #820

luismalta committed:
[ADD] fiscal_payment_ids onchange

luismalta committed:
[ADD] payment_term_id onchange

luismalta committed:
[FIX] financial_ids readonly

luismalta committed:
[ADD] compute change

mbcosta committed:
[12.0][MIG][REF] Inherited methods for merge, split and make extra stock moves.

mileo committed:
[FIX] Tests fiscal_payment

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[FIX] Required fields in payment

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mbcosta committed:
[12.0][MIG][IMP] Tests.

mileo committed:
[FIX] Document Payment term

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
WIP

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

gabrielcardoso21 committed:
[ADD] Filling of fiscal_payment_ids and financial_ids

mileo committed:
[FIX] Clean code payment in fiscal

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

hendrixcosta committed:
[FIX] Flake8

hendrixcosta committed:
[FIX] remove print

mileo committed:
[IMP] Demo, tests and usability

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[IMP] Test coverage of financial

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[FIX] lint errors [IMP] payment term

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[FIX] Confirmação do documento com financeiro

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

gabrielcardoso21 committed:
[FIX] Unlink of payment_term

renatonlima committed:
[FIX] field attribute

gabrielcardoso21 committed:
[ADD] Fill journal_id with operation's journal_id

gabrielcardoso21 committed:
[ADD] Create account.move after autorization

gabrielcardoso21 committed:
[ADD] Basic fields for account in account.chart.template

gabrielcardoso21 committed:
[ADD] Show account fields in view of chart

gabrielcardoso21 committed:
[ADD] Set accounts in template lines at install

gabrielcardoso21 committed:
[ADD] Load accounts for cfc chart template

gabrielcardoso21 committed:
[ADD] Load accounts for simple chart template

renatonlima committed:
Rename res.company to stock.picking.type

renatonlima committed:
pos rebase clean up

renatonlima committed:
[REF] get values for invoice creation

renatonlima committed:
[FIX] stock picking type

renatonlima committed:
[FIX] stock rule

renatonlima committed:
[FIX] create invoice from wizard

renatonlima committed:
[FIX] stock rule

renatonlima committed:
pep8

renatonlima committed:
remote debug command

renatonlima committed:
[FIX] view

renatonlima committed:
[FIX] view

renatonlima committed:
[REF] get journal from fiscal operation

mileo committed:
[REF] REM document_abstract.py

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[REF] Melhorias pos-rebase

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[FIX] document_abstract removal

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

fixup! [NEW] Template at document and document line

mileo committed:
[NEW] Company dependent account.move.template

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[FIX] pep8

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

renatonlima committed:
[FIX] default and prepare methods

gabrielcardoso21 committed:
[ADD] Load correctly both l10n_br_coa modules

mileo commented:
TODO:

- [ ] Postar lançamentos automaticamente;
- [ ] Gerar lançamentos automaticamente;

gabrielcardoso21 committed:
[REF] Set lines in template lines

mileo committed:
[REF] _onchange_fiscal_operation_id

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[REF] operation_line_id

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[REF] disabling invoice tests

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo commented:
@gabrielcardoso21 

Testei o PR localmente:

1) Os dois planos de contas estão sendo carregados corretamente, visto que o **Generic** não é carregado previamente.

Conforme conversamos na sexta feira, seria interessante criar dados de demonstração com uma empresa do **Lucro Presumido**, para tanto basta vc criar a empresa de forma similar a empresa do simples esta sendo criado e migrar todos os dados de demonstração da empresa do "base.main_company" a empresa criada.

Creio que é melhor vc fazer isso em um outro PR.

Alem disso, vc pode também alterar as permissões do usuário admin para ele já fazer parte do grupo de funcionalidades avançadas da contabilidade e alem disso definir que sua empresa principal é a do lucro presumido.

Isso vai facilitar muito os testes no runbot.

2) O diário não esta sendo preenchido nos dados de demonstração e nem quando a operação é trocada no documento fiscal.

gabrielcardoso21 committed:
[REF] Reuse common code

gabrielcardoso21 committed:
[ADD] Remove old accounting to load new chart template

gabrielcardoso21 committed:
[ADD] Post moves automatically

mileo commented on l10n_br_account/models/fiscal_document.py:
```diff
@@ -37,9 +37,20 @@ def _compute_move_template_ids(self):
+    def action_document_confirm(self):
+        super(FiscalDocument, self).action_document_confirm()
+        if self.journal_id.generate_moves:
+            self.action_move_create()
```
O lançamento contábil deve ser feito depois da autorização da nota.

gabrielcardoso21 replied:
Ok, estou corrigindo aqui

gabrielcardoso21 committed:
[ADD] Hide button if it cannot be pressed

gabrielcardoso21 committed:
[FIX] Function account_move_create

gabrielcardoso21 committed:
[REM] Conventional tax account moves for l10n_br_coa_simple

mileo commented:
![image](https://user-images.githubusercontent.com/1975978/82951244-2cd36000-9f7d-11ea-9889-b87df36f61fd.png)


![image](https://user-images.githubusercontent.com/1975978/82951258-31981400-9f7d-11ea-912e-69083fa2c874.png)

mileo commented:
![image](https://user-images.githubusercontent.com/1975978/82951318-4bd1f200-9f7d-11ea-81cb-3537e7251d26.png)

luismalta committed:
[ADD] History simples_nacional

luismalta committed:
[ADD] Template Line simples_nacional

luismalta committed:
[REF] Simple National field string

luismalta committed:
[ADD] simple_national_credit_id and simple_national_debit_id fields

luismalta committed:
[ADD] simple_national fields on hooks

luismalta committed:
[ADD] simple_national_credit_id and simple_national_debit_id data

mileo commented:
Por favor façam um rebase.

renatonlima committed:
Rename fiscal operation and operation line

renatonlima committed:
Update stock.rule method

renatonlima committed:
[MIGR] l10n_br_stock_account procurement.group

renatonlima committed:
Rename wizard folder

mileo committed:
[MOV] check/generate/onchange financial to l10n_br_fiscal.document.mixin.methods

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[IMP] DEMO DATA of fiscal payments

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[REM] Disable payment_order tests

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[REF] Move financial fields to fiscal mixin

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[NEW] Payment mixin

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[REM] Currency from account.move to avoid errors

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[NEW] PAyment mixin at account.invoice

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

renatonlima committed:
[REF] rename field

mileo committed:
[FIX] tax_framework coa install

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

renatonlima committed:
Update create invoice wizard

mileo committed:
[REF] Payment Condition / Payment Mode / Payment Term

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[FIX] Flake8, Lint and tests

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[FIX] Payment mode attrs

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

renatonlima committed:
[ADD] default stock fiscal operations

renatonlima committed:
[FIX] create invoice from picking wizard

renatonlima committed:
[REF] methods super calls

renatonlima committed:
[REF] views

renatonlima committed:
[FIX] stock picking and move views

renatonlima committed:
[FIX] picking return

mileo committed:
fixup! [NEW] PAyment mixin at account.invoice

mileo committed:
[REM] unused view

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[NEW] Grupo roteiros contabeis

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[REF] Grupo roteiros contabeis e workflow

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[NEW] Adapting l10n_br_fiscal payment mode

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[NEW] Adapting l10n_br_account payment mode

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[REM] Payment Condition

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[REM] Payment Mode

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[NEW] Create core account moves from fiscal financial ids

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[IMP] Financial mixin

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[FIX] Account payment and payment mode actions

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[NEW] Invoice id at fiscal document

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[NEW] Confirm edoc confirms invoice and vice versa

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[FIX] Invoice and Edoc Payment Sync

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[FIX] Template account move without date

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

jpcarassato commented:
Bom dia!

Efetuando o teste na base que o @mileo disponibilizou encontrei a seguinte situação:

Configurei a Operação com os impostos e criei a Sales Ordes e na mesma os calculos funcionou normalmente..
Como está dando erro ao criar a Invoice através da Sales Order, criei a Invoice pelo módulo Fiscal..
Porém na Invoice o Calculo dos impostos não funcionou


![image](https://user-images.githubusercontent.com/52328080/87042529-29610480-c1ca-11ea-8e5c-b33b4d1e1d89.png)

![image](https://user-images.githubusercontent.com/52328080/87042593-409ff200-c1ca-11ea-9088-b18542e74ece.png)

DiegoParadeda committed:
[ADD] MIS Reports for simples nacional

DiegoParadeda committed:
[ADD] mis-builder oca_dependency

renatonlima committed:
[ADD] demo

renatonlima committed:
pep8

mileo committed:
[FIX] Demo data of l10n_br_stock_account

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[NEW] Stock Company Fields

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

DiegoParadeda committed:
[ADD] l10n br website_sale

DiegoParadeda committed:
[FIX] Manifest

DiegoParadeda committed:
[FIX] Lint

DiegoParadeda committed:
[FIX] Author

DiegoParadeda committed:
[ADD] Cleave lib

DiegoParadeda committed:
[FIX] Pylint

DiegoParadeda committed:
[FIX] Use cleave from l10n_br_portal - add depends

DiegoParadeda committed:
[FIX] Pylint

mileo committed:
[FIX][WIP] Compute taxes on picking

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[IMP] Default operation main company demo

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[NEW] Comment on stock.picking

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[IMP] Propagate stock.rule fields to stock.move on pull rules

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[IMP] Demo data and tests

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[FIX] pep8

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[REM] Duplicated demo data

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[NEW] DRE auto expand accounts

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[NEW] OCA readme files

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

gabrielcardoso21 committed:
[ADD] Button to related Documents

gabrielcardoso21 committed:
[ADD] Load taxes in account_invoice_line

gabrielcardoso21 committed:
[MIG] Tests for invoice taxes

gabrielcardoso21 committed:
[FIX] Price

DiegoParadeda committed:
[FIX] Public zip search route

DiegoParadeda committed:
[FIX] get zip

DiegoParadeda committed:
[ADD] Tests

DiegoParadeda committed:
[FIX] Test assert

DiegoParadeda committed:
[FIX] city_id

gabrielcardoso21 committed:
[ADD] Field Invoice State to operation

gabrielcardoso21 committed:
[REF] Default fiscal operation

gabrielcardoso21 committed:
[REF] Invoice state related to operation

gabrielcardoso21 committed:
[REF] Position of fiscal operation in view

gabrielcardoso21 committed:
[ADD] Possibility to change quantity done

DiegoParadeda committed:
[ADD] Licencing info

gabrielcardoso21 committed:
[REF] Postion of field quantity_done

gabrielcardoso21 committed:
[FIX] Flake8

gabrielcardoso21 committed:
[FIX] Price in backorder

DiegoParadeda committed:
[ADD] Generated readme

gabrielcardoso21 committed:
[FIX] Requirements

DiegoParadeda committed:
[FIX] Zip and city_id required

gabrielcardoso21 committed:
[ADD] Load taxes and correct values in backorder

gabrielcardoso21 committed:
[FIX] Operation demo

gabrielcardoso21 committed:
[REF] Set line taxes before invoice creation

gabrielcardoso21 committed:
[REM] Wrong test

DiegoParadeda committed:
[ADD] CPF/CNPJ field

DiegoParadeda committed:
[ADD] Company type

mileo committed:
[NEW] account.account.type in a easy way to understand

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[REF] Re-classificate accounts of l10n_br_coa_simple

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[REF] Re-classificate accounts of l10n_br_coa_generic

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[FIX] Unused account.account.type

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[FIX] l10n_br_coa_simple wrong account.account.type

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[FIX] DRE expression CSLL e IRPJ

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[REF] BP with new account.account.types

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[NEW] account.account.type apuracao

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[FIX] account template generic

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[FIX] l10n_br_mis_report dependencies

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[FIX] account template simple

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

gabrielcardoso21 committed:
[FIX] Flake8

gabrielcardoso21 committed:
[REF] Operation always visible

gabrielcardoso21 committed:
[REM] Wrong function: stock.move doesn't have references to account.tax

gabrielcardoso21 committed:
[REF] Invoice State configurable only by operation

DiegoParadeda committed:
[ADD] CNPJ CPF Cleave

DiegoParadeda committed:
[FIX] Open form with city_id filled

DiegoParadeda committed:
[FIX] Public zip search with csrf enabled

DiegoParadeda committed:
[FIX] Flake 8

gabrielcardoso21 committed:
[REM] Obsolete button

DiegoParadeda committed:
[FIX] empty field city_id error

DiegoParadeda committed:
[FIX] cnpj_cpf validation and formatting

DiegoParadeda committed:
[FIX] Initiate form with city filled

DiegoParadeda committed:
[FIX] Empty city on form error

DiegoParadeda committed:
[FIX] Company type display

DiegoParadeda committed:
[FIX] Number and district form save

mileo committed:
[FIX

mileo committed:
[FIX] Mis Report

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[FIX] No update tax group

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[FIX] DRE e BP images

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

OCA-git-bot committed:
Merge PR #956 into 12.0

Signed-off-by rvalyi

oca-travis committed:
[UPD] Update l10n_br_stock_account.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

DiegoParadeda committed:
[FIX] Extend address mandatory fields instead of overwriting

DiegoParadeda committed:
[FIX] Street2 not mandatory

DiegoParadeda committed:
[ADD] Foreign user data validation and conditional fields

mileo committed:
[FIX] wrong bp expression

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

renatonlima committed:
[FIX] CST in

renatonlima committed:
Adapting demo data for multi company

renatonlima committed:
[ADD] l10n_br_sale hook to adapt multi company sale orders

renatonlima committed:
[FIX] demo user partner

OCA-git-bot committed:
Merge PR #1000 into 12.0

Signed-off-by rvalyi

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_stock_account
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_stock_account/

OCA-git-bot committed:
Merge PR #1001 into 12.0

Signed-off-by rvalyi

OCA-git-bot committed:
Merge PR #986 into 12.0

Signed-off-by rvalyi

mileo committed:
[FIX] Data data after rebase

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[REF] Remove payment from fiscal mixin

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

OCA-git-bot committed:
Merge PR #964 into 12.0

Signed-off-by mileo

oca-travis committed:
[UPD] Update l10n_br_hr.pot

oca-travis committed:
[UPD] Update l10n_br_website_sale.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[ADD] setup.py

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_hr/

mileo committed:
[FIX] Flake8

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[FIX] nfse financial tests

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

sbidoul committed:
[FIX] manifest website key

OCA-git-bot committed:
Merge PR #1005 into 12.0

Signed-off-by sbidoul

OCA-git-bot committed:
l10n_br_website_sale 12.0.1.0.1

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #798 into 12.0

Signed-off-by mileo

oca-travis committed:
[UPD] Update l10n_br_coa.pot

oca-travis committed:
[UPD] Update l10n_br_mis_report.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

OCA-git-bot committed:
l10n_br_coa 12.0.2.1.0

OCA-git-bot committed:
l10n_br_account 12.0.2.1.0

OCA-git-bot committed:
l10n_br_mis_report 12.0.1.1.0

OCA-git-bot committed:
l10n_br_coa_simple 12.0.2.1.0

OCA-git-bot committed:
l10n_br_coa_generic 12.0.2.1.0

OCA-git-bot committed:
[UPD] addons table in README.md

mileo committed:
[FIX] Runbot field with same label

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[IMP] Melhorias pos rebase

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>
